src/generator/gen_unicode.ml

```diff
@@ -177,9 +145,58 @@ let () =
   Printf.fprintf ch "(* Edit gen_unicode.ml.inc instead. *)\n\n";
   Printf.fprintf ch "\n\nlet version = %S\n\n" version;
   Printf.fprintf ch "module Categories = struct\n\n";
-  print_elements ch categories;
+  print_elements ch categories
+    [
+      "cc";
+      "cf";
+      "cn";
+      "co";
+      "cs";
+      "ll";
+      "lm";
+      "lo";
+      "lt";
+      "lu";
+      "mc";
+      "me";
+      "mn";
+      "nd";
+      "nl";
+      "no";
+      "pc";
+      "pd";
+      "pe";
+      "pf";
+      "pi";
+      "po";
+      "ps";
+      "sc";
+      "sk";
+      "sm";
+      "so";
+      "zl";
+      "zp";
+      "zs";
+    ];
```

Comment:
I would make this a constant to make it clear what lists we're considering.

Reply:
Are you proposing to lift the constant at toplevel ? I'm not sure what you have in mind that would make it clear (` what list we're considering`)

Reply:
done